raft.go

```diff
@@ -818,6 +819,7 @@ func (r *Raft) leaderLoop() {
 				groupReady = append(groupReady, e)
 				groupFutures[idx] = commitLog
 				lastIdxInGroup = idx
+				close(commitLog.committed)
```

Comment:
I went back and forth between a "safer" option with a nil check (similar to the future error channel) and a more direct approach with an unconditional `close`. I opted for the latter because I couldn't identify a case where the same `logFuture` would be committed twice, which, in that case, would likely lead to a bigger issue.